src/resources/min-lib.d.ts

```diff
@@ -0,0 +1,19 @@
+// Typescript compiler requires the types below to be defined.
```

Comment:
You're right I was confused by the fact that all d.ts files are passed to the compiler and as I redefine NodeList as parametrized type, the compilation of others d.ts file failed. 
Separate the tests based on the fact they use lib.d.ts or not is enough to avoid conflict